plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/drilldown/Item.js

```diff
@@ -43,9 +46,18 @@ Ext.define('NX.view.drilldown.Item', {
   },
 
   /**
+   * @public
    * Set the page to load when the breadcrumb segment associated with this drilldown item is clicked
    */
   setItemBookmark: function(bookmark, scope) {
     this.itemBookmark = (bookmark ? { obj: bookmark, scope: scope } : null);
+  },
+
+  /**
+   * @public
+   * Set the currently selected mode (corresponds with a card index)
```

Comment:
would prefer it we just called it 'setCardIndex' or something to indicate that we're indexing a list structure since 'mode' is pretty generic and the usage is specific